Tests/Marketplace/marketplace_services.py

```diff
@@ -2218,6 +2224,17 @@ def is_unified_integration(self, file_path: str):
         ])
 
 
+class PackIgnored(object):
+    ROOT_FILES = [Pack.SECRETS_IGNORE, Pack.PACK_IGNORE]
+    NESTED_FILES = {
+        PackFolders.INTEGRATIONS.value: ["README.md", "Pipfile", "Pipfile.lock", "_test.py", "commands.txt"],
+        PackFolders.SCRIPTS.value: ["README.md", "Pipfile", "Pipfile.lock", "_test.py"],
+        PackFolders.TEST_PLAYBOOKS.value: [],
```

Comment:
add a comment to indicate an empty list means the whole folder is ignored (in addition to the comment inside the function).